community/glossary.md

```diff
@@ -0,0 +1,79 @@
+# Glossary
+
+This is the SourceCred glossary of terms. While some of these terms can be
+easily defined with a search, the definitions here are specific to SourceCred.
+If you would like to suggest a change to a term or addition of a term,
+please [open an issue](https://github.com/sourcecred/docs/issues).
+
+
+## User Glossary
+
+The "User Glossary" is ideal if you want to understand SourceCred 
+from the perspective of using it in your project. For example, you
+might need to understand what terms SourceCred uses to define people
+or currency.
+
+### Champion
+
+A champion is an individual that starts an initiative with a determined promise
+to finish it. The individual generally takes charge of a well-scoped task, and
+provides clear direction and confidence about completing it. By calling someone
+a champion, you are placing faith in their ability as a leader and visionary.
+You can read more about champions [here](concepts/champion.md)
+
+#### References
+
+ - [@Beanow definition on discourse](https://discourse.sourcecred.io/t/about-champions-and-heroes/291)
+ - [SourceCred champion concept](concepts/champion.md)
+
+### Cred
+
+Cred is a metric that describes how important a contribution (or contributor) was to a particular project. Using SourceCred's own documentation as an example: the pull request that added this term to the glossary might have 10 cred, while a different pull request that fixes a typo might have only one cred. That would indicate that the first pull request was ten times as important. Cred "flows" between contributions and contributors; so for example, if you write a pull request that earns cred, some of that cred will flow to you.
+
+Cred is project-specific, and is computed based on the project's own [contribution graph](). For more details, see the [deep dive guide on cred](https://discourse.sourcecred.io/t/a-gentle-introduction-to-cred/20) or the [concept guide here](concepts/cred.md).
```

Comment:
Nit: link missing for contribution graph

Reply:
Just want to verify before I add it - this one? https://sourcecred.io/cred/timeline/@sourcecred/

Reply:
Yeah I think for now that's the best one :]
([New UI](https://discourse.sourcecred.io/t/sourcecred-beta-roadmap/422/3) pending)

Reply:
All set.